lib/logger/utils.ts

```diff
@@ -206,3 +206,46 @@ export function withSanitizer(streamConfig: bunyan.Stream): bunyan.Stream {
 
   throw new Error("Missing 'stream' or 'path' for bunyan stream");
 }
+
+/**
+ * A function that terminates exeution if the log level that was entered is
+ *  not a valid value for the bunyan logger.
+ * @param logLevelToCheck
+ * @returns returns undefined when the logLevelToCheck is valid. Else it stops execution.
+ */
+export function validateLogLevel(
+  logLevelToCheck: string | undefined | bunyan.LogLevel
+): void {
+  const allowedValues: bunyan.LogLevel[] = [
+    'trace',
+    'debug',
+    'info',
+    'warn',
+    'error',
+    'fatal',
+  ];
+  if (!logLevelToCheck) {
+    // when logLevel is undefined
+    return;
+  }
+  if (typeof logLevelToCheck === 'string') {
```

Comment:
```suggestion
  if (is.string(logLevelToCheck)) {
```

Also import `@sindresorhus/is`.